cli/src/cli.rs

```diff
@@ -1265,7 +1265,7 @@ fn process_show_account(
         use_lamports_unit,
     };
 
-    config.output_format.formatted_print(&cli_account);
+    println!("{}", config.output_format.formatted_string(&cli_account));
```

Comment:
We can use the same pattern here if we `formatted_string()` into a `mut String`, then change the `println!()`s below to `writeln!()`s.

Reply:
Ahhh, nice. I figured there was some way to fix this one, but didn't spend enough thoughts on it!

Reply:
✔️